src/cadence/mainnet/getNFTIDs.cdc

```diff
@@ -438,6 +439,11 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
     .borrow<&{SwaychainNFT.SwaychainNFTCollectionPublic}>() {
         ids["Swaychain"] = col.getIDs()
     }
+	
+	if let col = owner.getCapability(/public/MyNFTCollection)
```

Comment:
Also seeing this error:
    |
475 | 		ids["MyNFT"] = col.getIDs()
    |                      ^^^^^^ unknown member

Reply:
Applied the fix and updated files on my repo. 

Please review it.